src/github/entities/user-installation.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserInstallation extends Base {
+  @Column({ type: 'varchar', length: 20 })
+  installationId: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  organizationName: string;
+
+  @ManyToOne(() => User, (user) => user.installations)
+  user: User;
+}
```

Comment:
PullProofAI

The `UserInstallation` entity is missing validation for required fields.
```typescript
6: export class UserInstallation extends Base {
7:   @Column({ type: 'varchar', length: 20, nullable: false }) // Add nullable: false for validation
8:   installationId: string;
9: 
10:   @Column({ type: 'varchar', length: 255, nullable: false }) // Add nullable: false for validation
11:   organizationName: string;
12: 
13:   @ManyToOne(() => User, (user) => user.installations)
14:   user: User;
15: }
```


<!-- This is an auto-generated comment by PullProofAI -->